docs/docs/overview/quickstart/get-started-with-React.md

```diff
@@ -0,0 +1,230 @@
+});
+```
+
+You can get your API key from the Novu dashboard. Replace `YOUR_NOVU_API_KEY_HERE` with it. Now, if you’ll go to the Novu dashboard, you shall see the subscriber we created above with `subscriberId` of `123`.
+
+You can also update information about an already existing subscriber using the `subscriber.update` method as shown below:
+
+```jsx
+import { Novu } from '@novu/node';
+
+const novu = new Novu('<YOUR_NOVU_API_KEY>');
+
+await novu.subscribers.update('123', {
+  firstName: 'Saurabh', // new first name
+  lastName: 'Sumit', // new last name
+});
+```
+
+## Using Novu in a React app
+
+We have already installed the Novu notification center package above. To use it in an app, simply import it and add the component in your React app as follows:
+
+```jsx
+import React, { useState } from 'react';
+import {
+  NovuProvider,
+  PopoverNotificationCenter,
+  NotificationBell,
+} from '@novu/notification-center';
+import axios from 'axios';
+
+function App() {
+  const [description, setDescription] = useState('');
+  function onNotificationClick(message) {
+    // your logic to handle the notification click
+    if (message?.cta?.data?.url) {
+      window.location.href = message.cta.data.url;
+    }
+  }
+  const onClickHandler = async (e) => {
+    e.preventDefault();
+    const response = await axios.post('http://localhost:3000/sendNotif/create', { description });
+    setDescription('');
+    // console.log(response);
+  };
+
+  return (
+    <>
+      <NovuProvider
+        subscriberId={'<YOUR_SUBSCRIBER_ID>'}
+        applicationIdentifier={'<YOUR_NOVU_APPLICATION_IDENTIFIER>'}
+        initialFetchingStrategy={{
+          fetchNotifications: true,
+          fetchUserPreferences: true,
+        }}
+      >
+        <PopoverNotificationCenter
+          onNotificationClick={onNotificationClick}
+          listItem={(notification) => <div>{notification?.payload?.description}</div>}
+        >
+          {({ unseenCount }) => <NotificationBell unseenCount={unseenCount} />}
+        </PopoverNotificationCenter>
+      </NovuProvider>
+
+      <form onSubmit={onClickHandler}>
+        <input
+          placeholder="Enter notification text"
+          value={description}
+          onChange={(e) => setDescription(e.target.value)}
+        />
+        <button type="submit">Send</button>
+      </form>
+    </>
+  );
+}
+
+export default App;
+```
+
+You'll notice two things used above - `applicationIdentifier` and `subscriberID`.
+
+Application identifier is a public key used to identify your application. You can get your own application identifier from the [Novu dashboard settings](https://web.novu.co/settings).
+
+And subscribers are users to which notifications will be sent. They are identified by a `subscriberID` which you can also find in the [Novu subscribers dashboard](https://web.novu.co/subscribers).
+
+## Trigger a notification
+
+We can trigger a notification by simply using Novu’s trigger method as shown below:
```

Comment:
I find a bit confusing to introduce here the trigger of a notification using the Node package. Not sure if it is the right direction to go as might bias users to do it like this when they have other options to trigger the notifications (other SDKs, API, etc).
@unicodeveloper what do you think?

Reply:
@unicodeveloper Shall I move it to a codesandbox?

Reply:
@sumitsaurabh927 no need, you can use the multi-language template syntax here. So that it can show how to trigger notification from multiple languages and SDK

Reply:
But this quick start is for React. Would it still make sense or shall I let this be (to trigger notifs in React apps) and have a callout stating that it can be triggered across multiple languages and SDKs?

Reply:
Whether it's react, vue, angular, the triggering of notification happens from the backend. And it can be any backend. Node, PHP, Python etc.

Reply:
To make it simple, just move it to a codesandbox, or use an example from the ones we already have in codesandbox

Reply:
Done in latest commit!